src/sentry/api/endpoints/project_key_stats.py

```diff
@@ -30,35 +39,68 @@ def get(self, request: Request, project, key_id) -> Response:
         except ProjectKey.DoesNotExist:
             raise ResourceDoesNotExist
 
+        # Outcomes queries are coupled to Django's QueryDict :(
+        query_data = QueryDict(mutable=True)
+        query_data.setlist("field", ["sum(quantity)"])
+        query_data.setlist(
+            "outcome",
+            [
+                Outcome.ACCEPTED.api_name(),
+                Outcome.FILTERED.api_name(),
+                Outcome.RATE_LIMITED.api_name(),
+            ],
+        )
+        query_data["groupBy"] = "outcome"
+        query_data["category"] = "error"
+
+        now = timezone.now()
+        query_data["end"] = request.GET.get("until", now.isoformat())
+        query_data["start"] = request.GET.get("since", (now - timedelta(days=30)).isoformat())
```

Comment:
The former implementation used:
https://github.com/getsentry/sentry/blob/0440d5e346a04090d79f67116706daa4cebc9a8e/src/sentry/api/base.py#L433